src/Network/Network/Models/AzureFirewallPolicy/PSAzureFirewallPolicyRuleCollectionGroupWrapper.cs

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Network.Models;
+using MNM = Microsoft.Azure.Management.Network.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public class PSAzureFirewallPolicyRuleCollectionGroupWrapper
+    {
+
+        [JsonProperty(Order = 1)]
+        [Parameter(
```

Comment:
What is `[Parameter]` for?

Reply:
you are right. Its not needed. will fix this

Reply:
Fixed